src/content/guides/hot-module-replacement.mdx

```diff
@@ -399,5 +399,6 @@ There are many other loaders and examples out in the community to make HMR inter
 - [Elm Hot webpack Loader](https://github.com/klazuka/elm-hot-webpack-loader): Supports HMR for the Elm programming language.
 - [Angular HMR](https://github.com/gdi2290/angular-hmr): No loader necessary! A small change to your main NgModule file is all that's required to have full control over the HMR APIs.
 - [Svelte Loader](https://github.com/sveltejs/svelte-loader): This loader supports HMR for Svelte components out of the box.
+- [Live Reload Browser Page](https://github.com/Yuriy-Svetlov/live-reload-bp): If you enable the "Partial reload" option, the plugin will replace the module on the fly, without reloading the entire browser page.
```

Comment:
Which option are you referring to as the `Partial option`? I was not able to find any related option here - https://github.com/Yuriy-Svetlov/live-reload-bp#instance-options